wg-notebooks/charter.md

```diff
@@ -0,0 +1,39 @@
+# WG Notebooks Charter
+
+This charter adheres to the conventions, roles and organization management outlined in [wg-governance].
+
+## Scope
+
+WG Notebooks is responsible for the user experience around Notebooks and other IDEs and their integrations with Kubeflow.
+The WG covers launching and managing different types of Notebooks on Kubeflow as well as workflows that Notebook extensions enable.
+
+### In scope
+
+#### Code, Web apps, Controllers and Services
+
+- APIs and tools used for launching Notebooks (e.g. Notebooks CRD/Controller).
+- Web UI to manage and connect to Notebook instances (e.g. Notebook Manager UI)
+- SDKs and integrations from inside the Notebooks (e.g. Git inside Notebooks, Kale)
+- Documentation for detailed description of using Notebooks in Kubeflow.
+
+#### Cross-cutting and Externally Facing Processes
+
+- Coordinating with Control Plane WG to ensure that Notebooks manifests are properly deployed with Kubeflow.
+- Coordinating with Central Dashboard WG to ensure the integration with the Notebook Manager UI is up-to-date.
+- Coordinating with release teams to ensure that the Notebooks features can be released properly.
```

Comment:
Can you give an example for " Coordinating with release teams to ensure that the Notebooks features can be released properly" ? I am not sure I can wrap my head around what does it actually mean

Reply:
That the WG would communicate with the release teams and ensure that the artifacts are working and can be deployed without issues.

By Notebook features I was referring to the features we would develop around the Notebooks infrastructure [ web app, images, controller ]. The syntax could indeed be more clear here. 

Following the discussion [here](https://github.com/kubeflow/community/pull/407/files#r482682164) I'll remove this entry